packages/lab/src/input-next/InputNext.tsx

```diff
@@ -131,17 +102,6 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
 
   const [focused, setFocused] = useState(false);
 
-  const misplacedAriaProps = {
-    "aria-activedescendant": ariaActiveDescendant,
-    "aria-expanded": ariaExpanded,
-    "aria-owns": ariaOwns,
-    role,
-  };
-  const inputProps = mergeA11yProps(
```

Comment:
This used to merge the aria-labelledby props but this no longer happens?

Reply:
I moved that to line 156 as thought it was a bit clunky just to merge that, what do you think?